tool/config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "relaycheck": {
+    "targetnode": "http://127.0.0.1:8080",
+    "neighbor": "http://127.0.0.1:8080",

```

Comment:
What do you think if instead of only 1 neighbor we allow for multiple ones (they don't even need to be "neighbors")?
In that case, we could automatically check whether txs do get propagated over the network.